books/free-programming-books-langs.md

```diff
@@ -1926,6 +1926,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Learn to Program Using Python](https://web.archive.org/web/20201224032210/https://www.ida.liu.se/~732A47/literature/PythonBook.pdf) - Cody Jackson (PDF) *(:card_file_box: archived)*
 * [Learning to Program](http://www.alan-g.me.uk)
 * [Lectures on scientific computing with python](https://github.com/jrjohansson/scientific-python-lectures) - J.R. Johansson (2.7)
+* [Machine Learning with Python](https://www.tutorialspoint.com/machine_learning_with_python/machine_learning_with_python_tutorial.pdf) - Tutorials Point (PDF)
```

Comment:
This is already added [here](https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-subjects.md)